src/map/utils/charutils.cpp

```diff
@@ -1678,6 +1678,18 @@ namespace charutils
 
     void UnequipItem(CCharEntity* PChar, uint8 equipSlotID, bool update)
     {
+        if (PChar == nullptr)
+        {
+            ShowWarning("PChar was null.");
+            return;
+        }
+
+        if (equipSlotID < 0 || equipSlotID > 15)
```

Comment:
If equipSlotID is uint8 it is impossible for it to be less than zero (unsigned integer)